Makefile

```diff
@@ -18,27 +18,40 @@ CPPCHECK=ls lib/*.cc khmer/_khmermodule.cc | grep -v test | cppcheck -DNDEBUG \
 	 --quiet -Ilib -Ithird-party/bzip2 -Ithird-party/zlib \
 	 -Ithird-party/smhasher
 
-all: khmer/_khmermodule.so
 
+## all                  : default task; show list of build commands
+all: commands
+
+commands: Makefile
```

Comment:
I'm all for calling it `help` or `commands` or `tasks`...in fact, that is what I had done originally. I only changed it based on the suggestion in the original issue that the default `all` target be used, although I admit this would be counterintuitive to many.